internal/storage/postgres/treecluster/create.go

```diff
@@ -20,6 +21,7 @@ func defaultTreeCluster() *entities.TreeCluster {
 		Archived:       false,
 		LastWatered:    nil,
 		Trees:          make([]*entities.Tree, 0),
+		Name:           &name,
```

Comment:
So if you want to change the entities to a non-pointer value, you can use the string here as normal

```suggestion
		Name:           "",
```

Else, when you want to use a Pointer of a string in one line, I have created a utils function 

```go
utils.P("foo") // return &string
```